plugin/forward/fixtures/resolv.conf

```diff
@@ -0,0 +1,15 @@
+# This file is managed by man:systemd-resolved(8). Do not edit.
+#
+# This is a dynamic resolv.conf file for connecting local clients directly to
+# all known uplink DNS servers. This file lists all configured search domains.
+#
+# Third party programs must not access this file directly, but only through the
+# symlink at /etc/resolv.conf. To manage man:resolv.conf(5) in a different way,
+# replace this symlink by a static file or a different symlink.
+#
+# See man:systemd-resolved.service(8) for details about the supported modes of
+# operation for /etc/resolv.conf.
+
+nameserver 8.8.8.8
+nameserver 4.3.2.1
+search c.my-project.internal google.internal
```

Comment:
The comments and search domains in this file are likely pretty useless, but I left them in for authenticity. ;) Happy to strip them out.